### ISSUE TITLE
Calculate instantaneous (real-time) throughput from trace 

### ISSUE THREAD

**Issue by patrickkasper** [alibaba-edu/High-Precision-Congestion-Control#35]
I'm capturing trace files from all nodes, but calculating throughput of a flow does not seem to be easy since traces include packet level events. I have read the HPCC paper and it seems this is already done, so I was wondering is there any script available for it? I couldn't find anything in analysis folder. If not, what is the process to calculate flow throughput from the trace file so that I can script it?

Thanks in advance for your help!

**Comment by dongzhiydz** [comment]
Analyze whether the TR file can be solved

**Comment by dongzhiydz** [comment]
> I'm capturing trace files from all nodes, but calculating throughput of a flow does not seem to be easy since traces include packet level events. I have read the HPCC paper and it seems this is already done, so I was wondering is there any script available for it? I couldn't find anything in analysis folder. If not, what is the process to calculate flow throughput from the trace file so that I can script it?
> 
> Thanks in advance for your help!

Yes, there is no script to analyze the TR file. This script should be written by yourself. How do you analyze throughput
What? My email, welcome to communicate, ydz3116182132@163.com
